src/test/index.js

```diff
@@ -22,44 +31,91 @@ jest.mock( '@wordpress/react-native-editor/src/setup', () => ( {
 	default: jest.fn().mockReturnValue( <></> ),
 } ) );
 
-const initialProps = { initialData: '', capabilities: {} };
+const defaultLocale = 'en-gb';
+const setupLocaleLogs = [
+	[ 'locale', defaultLocale, getGutenbergTranslation( defaultLocale ) ],
+	[
+		'jetpack - locale',
+		defaultLocale,
+		getJetpackTranslation( defaultLocale ),
+	],
+	[
+		'layout-grid - locale',
+		defaultLocale,
+		getLayoutGridTranslation( defaultLocale ),
+	],
+];
 
-const initGutenbergMobile = ( props = initialProps ) => {
+const getEditorComponent = () => {
 	let EditorComponent;
 	AppRegistry.registerComponent.mockImplementation(
 		( name, componentProvider ) => {
 			EditorComponent = componentProvider();
 		}
 	);
-	jest.isolateModules( () => {
-		// Import entry point to initialize the editor
-		require( '../index' );
-	} );
-	return render( <EditorComponent { ...props } /> );
+	registerGutenbergMobile();
```

Comment:
After running several tests, I noticed that now using `isolateModules` when rendering the editor causes failures. As far as I investigated, it duplicates the Redux stores, in my case I verified it was creating two different stores for `@wordpress/blocks`. This leads to a scenario where blocks are registered into the second store but the editor complains about the lack of blocks (most likely because it's accessing the first one).

For this reason, I decided to stop using the workaround of isolating modules for registering Gutenberg on each test, and instead export the register function and call it here.